Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -156,20 +149,10 @@ def GetPackagesPath(self):
     # ####################################################################################### #
     #                         Actual Configuration for Platform Build                         #
     # ####################################################################################### #
-class PlatformBuilder( UefiBuilder, BuildSettingsManager):
+class PlatformBuilder(UefiBuilder, BuildSettingsManager):
     def __init__(self):
         UefiBuilder.__init__(self)
 
-    def CleanTree(self, RemoveConfTemplateFilesToo=False):

```

Comment:
This I think we should keep, but if it isn't getting called, we should call it when we do `--clean` so that TF-A is freshly built, also.

Reply:
I was mistaken. This is used when we call --clean. I'll add it back to the build.